pom.xml

```diff
@@ -106,15 +106,15 @@
     <beam.version>2.0.0</beam.version>
 
     <dataflow.release_name>Google Cloud Dataflow SDK for Java</dataflow.release_name>
-    <dataflow.container_version>${project.version}-20170317</dataflow.container_version>
+    <dataflow.container_version>${project.version}-20170517</dataflow.container_version>
     <dataflow.environment_major_version>6</dataflow.environment_major_version>
 
     <!-- Archetype version propagated automatically to underlying archetype modules.
          Example values are: ${project.version}, 1.2.3-SNAPSHOT, [1.0.0,2),
                              [${project.version}, 2), etc.
          Starter archetype reference copy isn't affected by this value; a manual
          change is expected there. -->
-    <archetype.sdk_version_dependency>2.0.0-beta4-SNAPSHOT</archetype.sdk_version_dependency>
+    <archetype.sdk_version_dependency>2.1.0-SNAPSHOT</archetype.sdk_version_dependency>
```

Comment:
Is there a reason this is not (as cited in the comment) `${project.version}` here and on the release branch? It seems like setting it thus would obviate the need for this manual change.